support/python/README.rst

```diff
@@ -0,0 +1,93 @@
+Relay Python SDK
+================
+
+This is the Python SDK for use with the internal `Relay <https://relay.sh>`_ service APIs.
+The SDK requires Python 3.8.
+
+It is intended for use by integration authors who are building containers to run 
+inside the service. For running workflows and interacting with the user-facing
+service APIs, use the `Relay CLI <https://github.com/puppetlabs/relay/>`_.
+
+The API documentation is auto-generated from the source code. Here are some
+higher-level examples that show the main SDK classes that integration authors
+will interact with.
+
+
+Installation
+------------
+
+The SDK is available to install via pip:
+
+.. code-block:: console
+
+  pip --no-cache-dir install "https://packages.nebula.puppet.net/sdk/support/python/v1/nebula_sdk-1-py3-none-any.whl"
+
+If you use the `relaysh/core:latest-python <https://hub.docker.com/r/relaysh/core/tags>`_ container image as your base
+image, it'll be pre-installed.
+
+Usage
+-----
+The main purpose of the SDK is to provide helpers for interacting with Relay's
+metadata service. Each container that runs in Relay has access to this service,
+which allows the container to read and write key-value data, emit events, and
+generate logs.
+
+Interface
+---------
+
+The `Interface class <./reference.html#module-nebula_sdk.interface>`_ is the primary way to interact with the SDK.
+Import it and instantiate an object, then call methods on that object to access metadata.
+Use the ``Dynamic`` class for getting data like connection credentials, 
+workflow-specific parameters, and secrets.
+
+.. code-block:: python
+
+  from nebula_sdk import Interface, Dynamic as D
+
+  relay = Interface()
+  param = relay.get(D.worflowparam)
+  secret = relay.get(D.mysecret)
+  relay.outputs.set("outputkey","This will be the value of outputkey")
+
+Webhooks
+--------
+
+The `Webhook class <./reference.html#module-nebula_sdk.webhook>`_ provides a
+helper that sets up a webserver to handle incoming requests for Trigger actions. 
+
+This example, from the `Dockerhub integration <https://github.com/relay-integrations/relay-dockerhub/>`_, makes use of
+the Interface class to access the ``events.emit`` method, which will cause
+the workflow associated with this trigger to be run with the inline mapping
+of workflow parameters to values extracted from the webhook payload.
+
+.. code-block:: python
+
+  from nebula_sdk import Interface, WebhookServer
+  from quart import Quart, request, jsonify, make_response
```

Comment:
It might be worth highlighting that the WebhookServer class can run any WSGI or ASGI application passed to it, and say that we like Quart for our own webhooks.

WSGI: https://www.python.org/dev/peps/pep-3333/
ASGI: https://asgi.readthedocs.io/en/latest/specs/main.html
Quart: https://pgjones.gitlab.io/quart/index.html